test/Test/BaseDAO/ProposalConfig.hs

```diff
@@ -34,12 +34,8 @@ config = DAO.defaultConfig
       push (2 :: Natural)
       swap
       ediv
-      ifSome
-        (do
-          unpair; dip drop
-        ) (do
-          push (0 :: Natural)
-        )
+      ifSome car $ do
+        push (0 :: Natural)
```

Comment:
`do` here looks redundant, I think it can be written as one line.

Reply:
Fixed this.